ref_ts/static/ref_ts/rickshaw/Makefile

```diff
@@ -1,95 +0,0 @@
-NODE_PREFIX=$(shell npm prefix)
```

Comment:
@zhiyuli If we are including JS libraries, surely we don't need to include their makefiles in the hydroshare repo, just the built products.